README.md

```diff
@@ -56,6 +56,10 @@ VSS execution is split into two phases: Phase I prioritizes recovery and single-
 multi-device use. Phase I is ready to use and integrated within [LDK-node]. Phase II will be subject to monitoring for
 demand from wallets and may slip to a later date. Refer [LDK-Roadmap] for more details.
 
+### API
+
+For detailed API contract of VSS, you can refer to the [VSS API contract].
```

Comment:
The sentence itself is still redundant. (i.e., it says the same thing twice with slightly different wording: "API contract of VSS" and "VSS API contract").

Let's word as: "See the [VSS API contract] for details."